dashboard/test/models/lti_integration_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class LtiIntegrationTest < ActiveSupport::TestCase
+  test "should validate required fields" do
+    assert_not build(:lti_integration, platform_id: nil).valid? "Missing platform_id should be invalid"
```

Comment:
Is the language here standard? I think "platform_id is required" is much more direct.